content/games/factions/rules.md

```diff
@@ -246,6 +246,23 @@ Well-Armed (1)
 |  Fortify   |  7+   |  18”  |    Another Group    | Until the start of your next turn. | Target may force enemies to reroll any number of to-hit dice once against them. |
 |  Counter   |  8+   |  18”  |        Group        |                 -                  | End any Spell effect on the target as if its duration ended.                    |
 
+#### Taunts
+
+| Taunt                          | Check | Requirements                                     | Effect                                                                                                                                 |
+| :----------------------------- | :---: | :----------------------------------------------- | :------------------------------------------------------------------------------------------------------------------------------------- |
+| Come and Die!                  |  6+   | -                                                | Cannot Move or Retreat (-1 FS if they would otherwise do so) and all enemy Groups are Reckless against them.                           |
+| We Are Unbreakable!            |  6+   | -                                                | Roll 3d6 and take the highest when testing Resolve but on failure, immediately Rout.                                                   |
+| We Leave No Friend Behind!     |  6+   | Must be within 6" of friendly Group              | While within 6" of a friendly Group re-roll any dice but lose 1d6 FS whenever a friendly Group within 6" Routs                         |
+| We Never Miss a Shot!          |  6+   | Groups with an MI profile only.                  | Must Shoot at the furthest enemy Group in range (if any) but may re-roll any misses when Shooting.                                     |
+| We'll Never Taste Your Blades! |  6+   | -                                                | Gain +1 T and +1 on to-hit rolls when Attacking but always count hits against them twice.                                              |
+| You'll Die with Us!            |  6+   | Must be Bloodied                                 | Inflict one hit for each taken when being Attacked but -1 T.                                                                           |
+| There Can Be Only One!         |  7+   | -                                                | Pick an enemy Group; when Attacking them gain +1 T and treat them as half as far away but must activate to Attack or Move toward them. |
```

Comment:
```suggestion
| There Can Be Only One!         |  7+   | -                                                | Pick an enemy Group. This Group must activate to Attack or Move toward that Group. When Attacking the chosen Group, gain +1 T, and treat this Group's MO as double its written value. |
```

Reply:
Is the second comma needed here, or should it be:
> When Attacking the chosen Group, gain +1 T and treat this Group's MO as double its written value.
I _think_ if we removed the conditional at the start we wouldn't have that second comma, right?
```suggestion
| There Can Be Only One! | 7+ | - | Pick an enemy Group. This Group must activate to Attack or Move toward that Group. When Attacking the chosen Group, gain +1 T and treat this Group's MO as double its written value. |
```